### PR TITLE
add click upgrade to hagrid ansible task

### PR DESCRIPTION
**Pull request by bitsofsteve**
## Description
This PR tends to fix an issue with Hagrid deploying to azure. During hagrid deploy the package click seems to be missing , even after the Hagrid setup ansible task already ran. This can cause the hagrid deploy to azure to fail. 

To reproduce. 

run `hagrid launch domain to azure --node-count 7` (the node count is totally optional, and go grab a beer or something)

Output (Don't bother with the IP's I already nuked them :))
Screenshot 2023-01-12 at 09 48 04

A bit of a digging(Thanks to @shubham3121 's support) shows that click seems to not have been installed. 
Screenshot from 2023-01-12 16-00-30

### CODE REVIEW

File: packages/grid/ansible/roles/containers/tasks/hagrid.yml [hunk 1]
```diff
@@ -8,3 +8,8 @@
   shell: "runuser -l {{ om_user }} -c 'pip install -e {{ syft_dir }}/packages/hagrid'"
   become: yes
   when: vagrant is defined and install == "true"
+
+- name: Install HAGrid
```
Comment (madhavajay) [inline packages/grid/ansible/roles/containers/tasks/hagrid.yml:12]: This is identical to the rule above and its also the wrong rule as this one will only run if `vagrant` is defined which we use in local Vagrant vm mode.

**Comment by shubham3121** [comment]
@bitsofsteve, found that the Click version on the azure machines is 7.0.0 which does not support `context_settings={"show_default": True}` as arguments on the `click.command` method. Versions > 7.1.0 support this argument. Upgrading the version of click should fix this.

File: packages/grid/ansible/roles/containers/tasks/hagrid.yml (hunk 1 above)
Comment (bitsofsteve) [reply packages/grid/ansible/roles/containers/tasks/hagrid.yml]: @madhavajay that was a weird text override.. was meant to be a click upgrade task.

**Comment by bitsofsteve** [comment]
> @bitsofsteve, found that the Click version on the azure machines is 7.0.0 which does not support `context_settings={"show_default": True}` as arguments on the `click.command` method. Versions > 7.1.0 support this argument. Upgrading the version of click should fix this. Screenshot from 2023-01-13 14-18-55

Yeah @shubham3121 , I tried pinning it, but it still wouldn't install a click version > 7.0.0

**Review by shubham3121 — APPROVED** [review]

File: packages/grid/ansible/roles/containers/tasks/hagrid.yml [hunk 2]
```diff
@@ -8,3 +8,7 @@
   shell: "runuser -l {{ om_user }} -c 'pip install -e {{ syft_dir }}/packages/hagrid'"
   become: yes
   when: vagrant is defined and install == "true"
+
```
Comment (madhavajay) [inline packages/grid/ansible/roles/containers/tasks/hagrid.yml:11]: Can we delete this now that its not needed.

File: packages/hagrid/setup.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
 
 packages = [
     "ascii_magic",
-    "click",
+    "click>7.1",
```
Comment (madhavajay) [inline packages/hagrid/setup.py:14]: We probably want >=7.1

**Review by madhavajay — CHANGES_REQUESTED** [review]
Two small changes and were good to go.

**Pull request merged**